contracts/OffsetHelper.sol

```diff
@@ -361,16 +365,6 @@ contract OffsetHelper is OffsetHelperStorage {
         // auto redeem pool token for TCO2; will transfer automatically picked TCO2 to this contract
         (tco2s, amounts) = PoolTokenImplementation.redeemAuto2(_amount);
```

Comment:
Let's start using `redeemAuto` here so we can eventually deprecate `redeemAuto2`.